examples/python/transformation/transform.py

```diff
@@ -0,0 +1,36 @@
+###############################################################################
+# (c) 2005-2015 Copyright, Real-Time Innovations.  All rights reserved.       #
+# No duplications, whole or partial, manual or electronic, may be made        #
+# without express written permission.  Any such copies, or revisions thereof, #
+# must display this notice unaltered.                                         #
+# This code contains trade secrets of Real-Time Innovations, Inc.             #
+###############################################################################
+
+"""Read a sample and write a new one."""
+
+from sys import path as sys_path
+from os import path as os_path
+
+file_path = os_path.dirname(os_path.realpath(__file__))
+sys_path.append(file_path + "/../../../")
+import rticonnextdds_connector as rti
+
+with rti.open_connector(
+    config_name = "MyParticipantLibrary::TransformationParticipant",
+    url = file_path + "/../ShapeExample.xml") as connector:
+
+    input = connector.get_input("MySubscriber::MySquareReader")
+    output = connector.get_output("MyPublisher::MyCircleWriter")
+
```

Comment:
Wait for matches + debug message?

Reply:
It depends how you want this to behave. Waiting for matches is optional. We're waiting for data anyway. I guess we could wait for a subscription before we start reading and transforming data, but then we would start caching data it in the reader.

I think the way it is, where it would transform live data and any late-joiner would get only what's new is OK.